### PR TITLE
Nuopc cmeps removed

### PR DESCRIPTION
**Pull request by jedwards4b**
Remove the nuopc driver and mediator to a separate repository https://github.com/ESCOMP/CMEPS.git

Test suite: scripts_regression_tests.py
Test baseline: 
Test namelist changes: 
Test status: bit for bit
Fixes 

User interface changes?: 

Update gh-pages html (Y/N)?:

Code review:

### CODE REVIEW

File: src/drivers/mct/cime_config/config_component_cesm.xml [hunk 1]
```diff
@@ -221,7 +221,7 @@
       <value compset="_CAM\d+%WCXI">288</value>
       <value compset="_DATM%COPYALL_NPS">72</value>
       <value compset="_DATM.*_CLM">48</value>
-      <value compset="_DATM.*_DICE.*_POP2">4</value>
+      <value compset="_DATM.*_DICE.*_POP2">24</value>
```
Comment (jedwards4b) [inline src/drivers/mct/cime_config/config_component_cesm.xml]: @mvertens is this change okay for mct?

**Comment by jgfouca** [comment]
@jedwards4b does this impact the problems @fischer-ncar found in the split PR?

**Comment by jedwards4b** [comment]
@jgfouca this is independent of the split PR, @billsacks is back and looking into it now.

File: src/drivers/mct/cime_config/config_component_cesm.xml (hunk 1 above)
Comment (mvertens) [reply src/drivers/mct/cime_config/config_component_cesm.xml]: no - can you please revert this to its original values. This was introduced in my testing for the nuopc branch and talking to Gustavo and Alper it seems that they want the original values kept. Thanks for checking.

**Pull request merged**